backend/src/main/java/hu/flowacademy/vajdasagbrand/controller/UserController.java

```diff
@@ -70,6 +70,12 @@ public void approveRegistration(@PathVariable("id") String userId) throws Valida
                 limit.orElse(defaultPageLimit));
     }
 
+    @RolesAllowed({"SuperAdmin", "CegAdmin"})
+    @GetMapping("/users/{id}")
+    public UserDTO getSingleUser(@PathVariable("id")String id) throws ValidationException {
+        return userService.getSingleUser(id);
```

Comment:
Nem vagyok teljesen képben, de szerintem a RolesAllowed-on túl még azért azt is ellenőrizni kellene, hogy a cégadmin csak a saját userét tudja lekérni, nem szerencsés, ha cégként más cégek adatait is ki tudom kérni...

Reply:
Szerintem ezt vegyük fel bug ticketben és most én mergelem így